### DISCUSSION TITLE
Is there a way to output the nlohmann type initializer for a json string?

### DISCUSSION THREAD

**Discussion by solaris783** [nlohmann/json#3063] [Q&A]
so a json string like:
{
	"Name": "John",
        "Surname": "Doe",
        "Colors": ['blue','red']
}

would be outputted as:
{
	{"Name", "John"},
        {"Surname", "Doe"},
        {"Colors",{'blue','red'}}
}

**Comment by nlohmann** [comment]
No, because that would be invalid JSON.

**Reply by solaris783** [reply]
The idea is to output a valid nlohmann::json initializer to use in projects, but I get why it sounds like a bit of an oddball request.  Thanks for responding.

**Reply by nlohmann** [reply]
I understand. Of course, you could write a function that traverses the DOM and creates the output you'd like to have.
